### PR TITLE
Add suggestions as made by Daniel E and increase call rate

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
Add suggestions as made by Daniel E in [this thread](https://centrepopgen.slack.com/archives/C01FBL6NWFN/p1664429696954009?thread_ts=1664149768.063269&cid=C01FBL6NWFN) and increase genotype call rate threshold, as per output from the [tob_wgs_call_rate_distribution.py](https://github.com/populationgenomics/tob-wgs/blob/main/scripts/validation/tob_wgs_call_rate_distribution.py) validation script (https://github.com/populationgenomics/tob-wgs/blob/main/scripts/validation/tob_wgs_call_rate_distribution.py) (see histogram [here](https://main-web.populationgenomics.org.au/tob-wgs/validation/tob_call_rate/v0/histogram_call_rate.png)).

### CODE REVIEW

File: scripts/eqtl_hail_batch/launch_eqtl_spearman.py [hunk 1]
```diff
@@ -783,6 +784,9 @@ def spearman_correlation(df):
     # for front-end analysis
     spearman_df = t.to_pandas()
     spearman_df['round'] = '1'
+    # change bp and round into int values
+    spearman_df['round'].astype(str).astype(int)
+    spearman_df['bp'].astype(str).astype(int)
```
Comment (lgruen) [inline scripts/eqtl_hail_batch/launch_eqtl_spearman.py]: Do you need two `astype` calls here (and below) or would `spearman_df['round'].astype(int)` work too?

File: scripts/eqtl_hail_batch/launch_eqtl_spearman.py [hunk 2]
```diff
@@ -1268,12 +1272,21 @@ def spearman_correlation(df):
     # turn into hail table and annotate with global bp and allele info
     t = hl.Table.from_pandas(adjusted_spearman_df)
     t = t.annotate(global_bp=hl.locus(t.chrom, hl.int32(t.bp)).global_position())
+    # add in vep annotation
```
Comment (lgruen) [inline scripts/eqtl_hail_batch/launch_eqtl_spearman.py:1275]: Was this change intentional? I'm asking because in the thread you said:

> For point 2, the functional annotation is there, it’s just not available for every variant

Does this change the annotation logic?

**Review by lgruen — CHANGES_REQUESTED** [review]

File: scripts/eqtl_hail_batch/launch_eqtl_spearman.py (hunk 1 above)
Comment (KatalinaBobowik) [reply scripts/eqtl_hail_batch/launch_eqtl_spearman.py:789]: I think you're right, it would work. I didn't take notes for this, unfortunately, but I remember some part breaking when I had `round` as an int, so I was just being precautious and doing it after since it didn't fail locally. Should be the same though (and I'll change this).

Comment (KatalinaBobowik) [reply scripts/eqtl_hail_batch/launch_eqtl_spearman.py]: I think you're right! I'll remove the redundant `astype(str)` bit

File: scripts/eqtl_hail_batch/launch_eqtl_spearman.py (hunk 2 above)
Comment (KatalinaBobowik) [reply scripts/eqtl_hail_batch/launch_eqtl_spearman.py:1275]: I'm just addressing @daniaki 's point (slack message [here](https://centrepopgen.slack.com/archives/C01FBL6NWFN/p1664429696954009?thread_ts=1664149768.063269&cid=C01FBL6NWFN)) about making the subsequent rounds of the eQTL tables the same:

> Would it be simple to modify the round*/sigsnps/*.parquet outputs by adding functional_annotation...

So here I'm just adding in functional annotation for rounds 2-5

Comment (lgruen) [reply scripts/eqtl_hail_batch/launch_eqtl_spearman.py:1275]: Ah, thanks for explaining!

**Review by lgruen — APPROVED** [review]
